src/borg/archiver.py

```diff
@@ -2866,6 +2867,17 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
         - The archive checks can be time consuming, they can be skipped using the
           ``--repository-only`` option.
 
+        The ``--max-duration`` option can be used to split a long-running repository check into multiple partial checks.
+        After the given number of seconds the check is interrupted. The next partial check will continue where the
+        previous one stopped, until 100% of the repository have been checked. Example: Running a check with
+        ``--max-duration=3600`` (one hour) every night that checks approx. 14.3% of the repository would mean one full
+        check per week.
+
+        Attention: Partial checks can only do way less checks than a full check (only the CRC32 checks on segment file
+        entries are done) and cannot be combined with ``--repair``. Partial checks may therefore be useful only with very
+        large repositories where a full check would last too long. Doing a full repository check resets the partial
+        check position to zero.
```

Comment:
in fact it even forgets that there was a ongoing partial check (it deletes the progress info from the repo config, not just resets it to zero). so maybe rather say:

... aborts / resets a partial check.